### DISCUSSION TITLE
Error "loading error: bundle bundle-msys.tar.gz: verification key not provided" occurred，when Load signed bundle in golang，

### DISCUSSION THREAD

**Discussion by woshizilong** [open-policy-agent/community#187] [OPA and Rego]
This is my go source sample.
```go
import "github.com/open-policy-agent/opa/rego"
......
query, err = rego.New(
		rego.LoadBundle("bundle-msys.tar.gz"),
		rego.Query("x = data.main.allow"),
		rego.Compiler(compiler),
		// rego.SkipBundleVerification(true),
	).PrepareForEval(ctx)
```

If make `rego.SkipBundleVerification(true)` available, no error will occur.But can't verify the bundle anymore.
How should I provide a valid certificate for verification?

**Comment by anderseknert** [comment]
Hi @woshizilong ! Did you look into using the [SDK](https://www.openpolicyagent.org/docs/latest/integration/#sdk) for this? That should provide you with all the capabilities of the OPA server, including loading a config file for this purpose.

**Reply by woshizilong** [reply]
@anderseknert 
Thanks for your answer.
I have used the sdk and it works fine. But there are many more packages imported. I want to reduce external dependencies.
